vms/avm/environment_test.go

```diff
@@ -221,6 +233,30 @@ func setup(tb testing.TB, c *envConfig) *environment {
 	return env
 }
 
+func staticConfig(tb testing.TB, f fork) config.Config {
+	var (
+		durangoTime  = mockable.MaxTime
+		eUpgradeTime = mockable.MaxTime
+	)
+
+	switch f {
+	case eUpgrade:
+		eUpgradeTime = time.Time{}
+		fallthrough
+	case durango:
+		durangoTime = time.Time{}
+	default:
+		require.NoError(tb, fmt.Errorf("unhandled fork %d", f))
```

Comment:
I didn't notice this in the previous PRs... We shouldn't be abusing `require.NoError` for this. Some variant of `require.FailNow` should definitely be used here instead. (can we clean this up in the other locations too?)